ConwaysGameOfLife.Tests/GeneratorShould.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Xunit;
+
+namespace ConwaysGameOfLife.Tests
+{
+    public class GeneratorShould
+    {
+        Generator _generator;
+        public GeneratorShould()
+        {
+            _generator = new Generator(new Universe(3, 3));
+        }
+        [Fact]
+        public void WhenInitialized_DisplayUniverse()
+        {
+            var output = new StubOutput();
+            var formatter = new OutputFormatter();
+            var expected = ". . . \n. . . \n. . . \n";
+
+            _generator.DisplayUniverse(output, formatter);

```

Comment:
What is the responsibility of the Generator? Should it be displaying the Universe that it generates?